core/Plugin/Menu.php

```diff
@@ -30,16 +30,6 @@
  */
 class Menu
 {
-    protected $module = '';
```

Comment:
It is called quite often and would be nice to have it just as fast as possible. It won't be a huge regression for sure